content/talks/responsive-components.md

```diff
@@ -26,10 +26,16 @@ press:
     url: https://twitter.com/schalkventerdev/status/1390205495110246400
     slug: shaping-css
 events:
+  - venue: Smashing Meets CSSummer
+    url: https://smashingconf.com/meets-css/
+    date: 2021-07-08
+    adr: Online
   - venue: Front Range Front-End
     url: https://www.meetup.com/front-range-front-end/events/kvckcsyccjbfb/
     date: 2021-06-03
     adr: Online
+    media: &frfe
+      iframe: https://www.youtube.com/embed/GuMvZuUrJaY?start=294
```

Comment:
@mirisuzanne Was it intentional for this to link to a spot in the middle of the talk, instead of the beginning?

Reply:
Did I intentionally add a start time? yes
Did I intentionally get it wrong? no
What was I thinking? 🙈

```suggestion
      iframe: https://www.youtube.com/embed/GuMvZuUrJaY
```